pyodide_build/out_of_tree/venv.py

```diff
@@ -95,6 +95,7 @@ def get_pip_monkeypatch(venv_bin: Path) -> str:
         ],
         capture_output=True,
         encoding="utf8",
+        check=False,
```

Comment:
In the case for this file, we might be able to omit this because we validate the result using `check_result()` above and add a logger message.

Reply:
```suggestion
        check=True,
```